src/app/components/Video.js

```diff
@@ -7,6 +7,12 @@ export default class Video extends Component {
       src: PropTypes.string.isRequired,
       type: PropTypes.string.isRequired,
     })).isRequired,
+    showMuteButton:PropTypes.bool.isRequired
+  };
+
+  static defaultProps = {
+    showMuteButton: true,
+    autoPlay: false,
```

Comment:
Default value should be `true` to make sure this refactor is compatible with my landing page.

Reply:
Also add `autoPlay: PropTypes.bool` to `propTypes`